### DISCUSSION TITLE
Cesium.SceneTransforms.wgs84ToWindowCoordinates is not a function

### DISCUSSION THREAD

**Discussion by sun2430987** [zouyaoji/vue-cesium#614] [Q&A]
cesium最新版本已不在支持Cesium.SceneTransforms.wgs84ToWindowCoordinates
# 1.121版本后，SceneTransforms.wgs84ToWindowCoordinates已被删除，需要用SceneTransforms.worldToWindowCoordinates代替。

**Comment by zouyaoji** [comment]
更新最新版哈
